spec/controllers/users_controller_spec.rb

```diff
@@ -45,7 +51,32 @@
       end
     end
 
-    context "with invalid input" do
+    context "valid personal info and declined card" do
+      
+      it "does not create a new user record" do
+        charge = double(:charge, successful?: false, error_message: "Your card was declined.")
+        StripeWrapper::Charge.should_receive(:create).and_return(charge)
+        post :create, user: Fabricate.attributes_for(:user), stripeToken: '1231241'
+        expect(User.count).to eq(0)
+      end
+
+      it "renders the new template" do
+        charge = double(:charge, successful?: false, error_message: "Your card was declined.")
+        StripeWrapper::Charge.should_receive(:create).and_return(charge)
```

Comment:
You should use the `expect` syntax as `should` syntax is not recommended anymore.

```ruby
expect(StripeWrapper::Charge).to receive(:create).and_return(charge)
```